app/models/task.py

```diff
@@ -1,6 +1,17 @@
 from flask import current_app
 from app import db
-
+from datetime import datetime
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.id'))
+    goal = db.relationship("Goal", back_populates="tasks")
+    
+
+    # def to_string(self):
+    #     return f"{self.id}: {self.title} {self.description} {self.completed_at}"
```

Comment:
YASSSS I was going to comment and say you could create an instance method here and use it in your code to reduce redundancy. Here is a suggestion on how it could look
```suggestion
    def to_json(self):
        if self.completed_at:
            completed = True
        else:
            completed = False

        return {
            "id": self.task_id,
            "title": self.title,
            "goal_id": self.goal_id,
            "description": self.description,
            "is_complete": completed,
        }
```